game/server/gameconfiguration.h

```diff
@@ -0,0 +1,37 @@
+abstract_class IGameConfiguration : public IAppSystem
+{
+    virtual int         GetMaxSplitscreenPlayers( void ) = 0;
+    virtual void        AllocateScratchRenderTargets( SwapChainHandle_t, const RenderDeviceInfo_t &, RenderMultisampleType_t ) = 0;
+    virtual void        *GetDesiredRenderingPipeline( void ) = 0;
+    virtual bool        InstantReplayShouldSkipMessage( int ) = 0;
+    virtual bool        InstantReplayShouldIgnoreMessage( int ) = 0;
+    virtual bool        ValidateNetworkVersion( int ) = 0;
+    virtual void        ChangeWorldMountNameByNetworkVersion( const char *, int ) = 0;
+    virtual const char *GetEncryptionKey( void ) = 0;
+    virtual void       *FindProceduralEntity( const char * ) = 0;
+    virtual void        SetupGameSessionConfiguration( KeyValues *, GameSessionConfiguration_t * ) = 0;
+    virtual void        InitGameSession( const GameSessionConfiguration_t & ) = 0;
+    virtual void        ShutdownGameSession( void ) = 0;
+    virtual const char *GetModName( void ) = 0;
+    virtual bool        ShouldPreserveEntityForRespawn( const char * ) = 0;
+    virtual void        LoopPostInitAllGameSystems( const GameSessionConfiguration_t &, ILoopModePrerequisiteRegistry * ) = 0;
+    virtual void        RegisterGameActivities( void ) = 0;
+    virtual void        ExecuteURL( const char * ) = 0;
+    virtual void        *GetBugReportLaunchURLParameters( KeyValues * ) = 0;
+    virtual void        *GetStringTableDefs( CUtlVector<StringTableDef_t> & ) = 0;
+    virtual bool        IsUserMessage( int ) = 0;
+    virtual KeyValues   *GetCurrentGameSessionConfigKeyValues( void ) = 0;
+.data.rel.ro:0000000003FFC8D0                 dq offset _ZN18CGameConfiguration18GetGameDescriptionEv ; CGameConfiguration::GetGameDescription(void)

```

Comment:
Potentially needs a cleanup?

Reply:
Whoops. Didn't mean to include that file. Removing...